src/pages/FuturePage.js

```diff
@@ -8,7 +8,7 @@ export default function FuturePage({ trips, onDeleteCard }) {
       <ul>
         {trips.map(trip => (
           <li key={trip._id}>
-            <FutureTripCard trip={trip} onDeleteCard={onDeleteCard} />
+            <FutureTripCard trip={trip} onDelete={onDeleteCard} id={trip._id} />
```

Comment:
You could just pass the `id` here instead of your card though:
```suggestion
            <FutureTripCard trip={trip} onDelete={() => onDeleteCard(trip._id)} />
```
Then you don't need to add the `id` as prop. And in your `FutureTripCard` you can just call `onDelete`:
```js
        <Modal
          onDelete={onDelete}
```